socorro/external/postgresql/staticdata.py

```diff
@@ -172,15 +172,6 @@ class ReportPartitionInfo(BaseTable):
             'date_processed',
             'TIMESTAMPTZ'
         ),
-        (
-            'processed_crashes',
-            '12',
-            '{uuid}',
-            '{date_processed}',
-            '{}',
-            'date_processed',
-            'TIMESTAMPTZ'
-        ),
```

Comment:
I'm removing the section from the `report_partition_info` section in static data which only affects the local dev environment. This PR doesn't have a migration to remove the tables or the `report_partition_info` row from -stage and -prod.

I'm on the fence about whether to include that in this PR or whether to push it off to a future PR.

Thoughts?

Reply:
IMO including it would be fine.